### ISSUE TITLE
Preference to disable on boot

### ISSUE THREAD

**Issue by paragbaxi** [keycastr/keycastr#299]
I use this for recording videos. I don't like that I have to disable it every time I reboot. I would prefer an option to enable/disable on boot.

**Comment by akitchen** [comment]
I don't think this is something we'll do, the potential for confusion is too high.
